From a81a4ad0f1bb55a2a4e60b88be7ddab17dc60771 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 16 Apr 2021 12:52:26 +0200
Subject: [PATCH 1/3] host-pip-requirements: add Jinja2

Version 2.10 is right now specified by Foris and as we will drop in the
future, let's use it here as well. Otherwise, I'd try to keep it updated
as much as I can

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/python/host-pip-requirements/Jinja2.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 lang/python/host-pip-requirements/Jinja2.txt

diff --git a/lang/python/host-pip-requirements/Jinja2.txt b/lang/python/host-pip-requirements/Jinja2.txt
new file mode 100644
index 000000000..4735c06c6
--- /dev/null
+++ b/lang/python/host-pip-requirements/Jinja2.txt
@@ -0,0 +1 @@
+Jinja2==2.10 --hash=sha256:f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4
-- 
2.25.1

